### DISCUSSION TITLE
Detection confidence score

### DISCUSSION THREAD

**Discussion by AlexPasqua** [PaddlePaddle/PaddleOCR#14009] [Q&A]
Hello, during inference, I see the result usually contains a confidence score. I looked into the code, and it seems to be the **recognition** confidence score. Is the confidence of the **detection** part accessible somewhere?
Thanks :)

**Comment by GreatV** [comment]
please refer to:

https://github.com/PaddlePaddle/PaddleOCR/blob/e621d034b584fae03c22cd51c26f6a52d62417b6/ppocr/postprocess/db_postprocess.py#L109-L158
